modules/gitops-disabling-the-creation-of-the-default-cluster-roles-for-the-cluster-scoped-instance.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assembly:
+//
+// * declarative_clusterconfig/customizing-permissions-by-creating-user-defined-cluster-roles-for-cluster-scoped-instances.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-disabling-the-creation-of-the-default-cluster-roles-for-the-cluster-scoped-instance_{context}"]
+= Disabling the creation of the default cluster roles for the cluster-scoped instance
+
+To add or remove permissions to cluster-wide resources, as needed, you must disable the creation of the default cluster roles for the cluster-scoped instance by editing the `ArgoCD` custom resource (CR) YAML file.
+
+.Procedure
+
+. Set the value of the `.spec.defaultClusterScopedRoleDisabled` field to `true`:
++
+.Example Argo CD custom resource
+
+[source,yaml]
+----
+apiVersion: argoproj.io/v1beta1
+kind: ArgoCD
+metadata:
+  name: example <1>
+  namespace: spring-petclinic <2>
```

Comment:
```suggestion
  namespace: spring-petclinic # <2>
```

Also, should this value be a replaceable (e.g. `namespace: <namespace_name>`)? I think users will recognize that `spring-petclinic` is just an example, but making it a replaceable makes it even more obvious.